node/external/nodeApiResolver_test.go

```diff
@@ -35,6 +35,7 @@ func TestNewNodeApiResolver_ShouldWork(t *testing.T) {
 
 	assert.NotNil(t, nar)
```

Comment:
useless assert (it is checked by `assert.False(t, nar.IsInterfaceNil())`)

Reply:
done.